/* -*-mode:c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */

package KohoProtobufs;

message RoamingPacket {
    required uint64 client_id = 1;
    required bytes body = 2;
}
